clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/model/KubernetesV2ServerGroup.java

```diff
@@ -159,6 +160,10 @@ public static KubernetesV2ServerGroup fromCacheData(CacheData cd, List<CacheData
     return new KubernetesV2ServerGroup(manifest, cd.getId(), instances, loadBalancers);
   }
 
+  public static KubernetesV2ServerGroup fromCacheData(ServerGroupCacheData cacheData) {
```

Comment:
If you're adding this to cleanup the number of parameters, can you apply it every place the fully expanded call gets used?

Reply:
Sure, though it was more so that the interface could have one type parameter.